package.json

```diff
@@ -48,12 +48,11 @@
     "yargs": "^16.2.0"
   },
   "main": "lib/ts/index.js",
+  "types": "lib/ts/index.d.ts",
   "files": [
-    "/bench",
     "/build",
     "/deployments",
-    "/lib",
+    "/lib/src",
```

Comment:
Why this change? Doesn't this cause the main (index.js) to no longer be packaged?

Reply:
Currently lib looks like:
```
lib
├── bench
├── src
│   └── index.js
└── test
```

I just thought the test and bench files were for development and didn't need to be included in the package. I can revert if you think there  is value in leaving them in. (Note that the deployment scripts are under `lib/src`).

Reply:
I'm just confused about line 50:

>"main": "lib/ts/index.js",

How is that included in the npm package if its path is not in the files list?

Reply:
Good catch! I didn't even notice. Looks like its pointing to the wrong "main". It pointing to here <https://unpkg.io/browse/@gnosis.pm/gp-v2-contracts@0.0.1-alpha.2/lib/ts/index.js> which gives you a `Not Found` but should point to:
<https://unpkg.io/browse/@gnosis.pm/gp-v2-contracts@0.0.1-alpha.2/lib/src/ts/index.js>

Reply:
Created #265

Reply:
I think I changed the `files` key in #259 and that made it into `main` branch :see_no_evil:. I'll resolve this thread and we can discuss in #265 as that PR is more focused on fixing the npm packaging parameters.